tests/Integration/Schema/Directives/Fields/CreateDirectiveTest.php

```diff
@@ -212,4 +212,64 @@ public function itCreatesAnEntryWithDatabaseDefaultsAndReturnsItImmediately()
             'default_string' => \CreateTestbenchTagsTable::DEFAULT_STRING,
         ], Arr::get($result, 'data.createTag'));
     }
+
+    /**
+     * @test
+     */
+    public function itDoesNotCreateWithFailingRelationship()
+    {
+        $schema = '
+        type Task {
+            id: ID!
+            name: String!
+        }
+        
+        type User {
+            id: ID!
+            name: String
+            tasks: [Task!]! @hasMany
+        }
+        
+        type Mutation {
+            createUser(input: CreateUserInput!): User @create(flatten: true)
+        }
+        
+        input CreateUserInput {
+            name: String
+            tasks: CreateTaskRelation
+        }
+        
+        input CreateTaskRelation {
+            create: [CreateTaskInput!]
+        }
+        
+        input CreateTaskInput {
+            name: String
+            user: ID
+        }
+        ' . $this->placeholderQuery();
+        $query = '
+        mutation {
+            createUser(input: {
+                name: "foo"
+                tasks: {
+                    corruptField: [{
+                        name: "bar"
+                    }]
+                }
+            }) {
+                id
+                name
+                tasks {
+                    id
+                    name
+                }
+            }
+        }
+        ';
+        $result = $this->execute($schema, $query);
```

Comment:
adding some newlines here can not hurt

Reply:
PHP style fixer is complaining if it is added

Reply:
PHP style fixer (php-cs-fixer) is complaining if it is added

Reply:
Single newlines are fine